otlp/scripts/generate-modules.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
Please update the README in this directory to explain how to use this.

Reply:
Done, and feel free to update it :)